src/applications/vaos/containers/ConfirmedAppointmentListPage.jsx

```diff
@@ -16,6 +16,13 @@ export class ConfirmedAppointmentListPage extends React.Component {
   }
   render() {
     const { appointments, status } = this.props;
+    const scheduleButton = (
+      <Link to="new-appointment">
+        <button type="button" className="usa-button" name="newAppointment">
```

Comment:
Looks good to me.  If you add the following 2 classes to this button, I think it will help a little bit with the spacing: `vads-u-margin-x--0 vads-u-margin-bottom--1p5`

Reply:
@narin , added the 2 classes as recommended